lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ConvertLocationAndScaleParametersToProbabilities.py

```diff
@@ -112,40 +114,43 @@ def setUp(self):
         self.template_cube = (
             set_up_probability_above_threshold_temperature_cube())
         self.template_cube = iris.util.squeeze(self.template_cube)
-        self.means = self.template_cube[0, :, :].copy()
-        self.means.units = 'Celsius'
-        self.variances = self.template_cube[0, :, :].copy()
-        self.variances.units = 'Celsius2'
+        self.location_parameters = self.template_cube[0, :, :].copy()
+        self.location_parameters.units = 'Celsius'
+        self.scale_parameters = self.template_cube[0, :, :].copy()
+        self.scale_parameters.units = 'Celsius2'
 
     def test_compatible_units(self):
         """Pass in compatible cubes that should not raise an exception. No
         assert statement required as any other input will raise an
         exception."""
-        Plugin()._check_unit_compatibility(self.means, self.variances,
-                                           self.template_cube)
+        Plugin()._check_unit_compatibility(
+            self.location_parameters, self.scale_parameters,
+            self.template_cube)
 
     def test_convertible_units(self):
         """Pass in cubes with units that can be made equivalent by modification
         to match the threshold units."""
-        self.means.units = 'Fahrenheit'
-        self.variances.units = 'Fahrenheit2'
-        Plugin()._check_unit_compatibility(self.means, self.variances,
-                                           self.template_cube)
-        self.assertEqual(self.means.units, "Celsius")
+        self.location_parameters.units = 'Fahrenheit'
+        self.scale_parameters.units = 'Fahrenheit2'
+        Plugin()._check_unit_compatibility(
+            self.location_parameters, self.scale_parameters,
+            self.template_cube)
+        self.assertEqual(self.location_parameters.units, "Celsius")
 
     def test_incompatible_units(self):
         """Pass in cubes of incompatible units that should raise an
         exception."""
-        self.means.units = 'm s-1'
+        self.scale_parameters.units = 'm s-1'
         msg = 'This is likely because the mean'
         with self.assertRaisesRegex(ValueError, msg):
-            Plugin()._check_unit_compatibility(self.means, self.variances,
-                                               self.template_cube)
+            Plugin()._check_unit_compatibility(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _check_unit_compatibility of a client class](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4576393)